#!/usr/bin/env python

PACKAGE = "fcc"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("segmentTh_H",           double_t, 0, "Default: 0.35",    0.35,    0,  2)
gen.add("segmentTh_V",           double_t, 0, "Default: 0.6",     0.6,    0,  2)

exit(gen.generate(PACKAGE, "fcc", "fcc_"))